The CI failure is due to the bug that this PR is attempting to fix:

```
curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh
golangci/golangci-lint info checking GitHub for latest tag
golangci/golangci-lint info found version: 1.17.1 for v1.17.1/linux/amd64
make: *** [setup] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,9 @@ setup: ## Install all the build and lint dependencies
 	mkdir -p bin
 	curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | sh
 	curl -sfL https://install.goreleaser.com/github.com/gohugoio/hugo.sh | sh
-	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh
+	# FIXME golangci-lint.sh is currently broken, restore installation with curl when it is fixed
+	# curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh
+	( cd $$(mktemp -d) && go mod init tmp && go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.17.1 )
 ifeq ($(OS), Darwin)
 	curl -sfL -o ./bin/shellcheck https://github.com/caarlos0/shellcheck-docker/releases/download/v0.4.6/shellcheck_darwin
 else

